src/openrct2/paint/tile_element/Surface.cpp

```diff
@@ -559,7 +559,7 @@ static void viewport_surface_draw_tile_side_bottom(paint_session * session, enum
     if (!is_csg_loaded() && edgeStyle >= TERRAIN_EDGE_RCT2_COUNT)
         edgeStyle = TERRAIN_EDGE_ROCK;
 
-    registers regs{};
+    sint16 al, ah, cl, ch, dl = 0, dh;
```

Comment:
All of the variables in this function can be named. No point wasting time refactoring to another interim variable name.